src/Items/ItemHandler.cpp

```diff
@@ -216,6 +217,11 @@ cItemHandler * cItemHandler::CreateItemHandler(int a_ItemType)
 		}
 
 		case E_ITEM_BEETROOT_SEEDS:
+		case E_BLOCK_CACTUS:
+		{
+			return new cItemCactusHandler(a_ItemType);
```

Comment:
This causes beetroot seeds to have a Cactus handler as well, is that intended?

Reply:
Nope, thanks